iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -244,11 +252,16 @@ public void renameTable(TableIdentifier from, TableIdentifier originalTo) {
   @Override
   public void createNamespace(Namespace namespace, Map<String, String> meta) {
     Preconditions.checkArgument(
-        !namespace.isEmpty(),
-        "Cannot create namespace with invalid name: %s", namespace);
-    Preconditions.checkArgument(isValidateNamespace(namespace),
-        "Cannot support multi part namespace in Hive Metastore: %s", namespace);
-
+        !namespace.isEmpty(), "Cannot create namespace with invalid name: %s", namespace);
+    Preconditions.checkArgument(
+        isValidateNamespace(namespace),
+        "Cannot support multi part namespace in Hive Metastore: %s",
```

Comment:
Search in that class for "Metastore" has a few typos. Not in upstream iceberg

Reply:
Sorry, I don't get it. Could you please show me an example of the typo?

Reply:
It was waaay older than this port. Anyway, I fixed it.

Reply:
Fixed.